### PR TITLE
package linode during build

### PR DESCRIPTION
**Pull request by kinarashah**
https://github.com/rancher/rancher/issues/21300

### CODE REVIEW

**Comment by alena1108** [comment]
@kinarashah can you check if we need to put changes for it to work with https://github.com/rancher/rancher/pull/21745

**Comment by kinarashah** [comment]
@alena1108 Updated PR to use the correct uiURL. Now, UI would always fetch from the local files `/assets/rancher-ui-driver-linode/component.js`, unless the driver's custom URL is changed. 
Don't think I need any changes for proxy/whitelisting domains for the linked PR because they're already restrictive enough, `"linode.github.io", "api.linode.com"`

File: package/Dockerfile [hunk 1]
```diff
@@ -61,6 +65,12 @@ VOLUME /var/log/auditlog
 RUN mkdir -p /usr/share/rancher/ui && \
     cd /usr/share/rancher/ui && \
     curl -sL https://releases.rancher.com/ui/${CATTLE_UI_VERSION}.tar.gz | tar xvzf - --strip-components=1 && \
+    cd assets && \
+    mkdir -p rancher-ui-driver-linode && \
```
Comment (dramich) [inline package/Dockerfile]: use `mkdir -p assets/rancher-ui-driver-linode && \`
then `cd assets/rancher-ui-driver-linode` just in case assets isn't there

**Comment by dramich** [comment]
lgtm

**Pull request merged**